app/controllers/mixins/actions/vm_actions/resize.rb

```diff
@@ -54,7 +54,12 @@ def resize_vm
                 flavor_id = params['flavor_id']
                 flavor = find_record_with_rbac(Flavor, flavor_id)
                 old_flavor_name = @record.flavor.try(:name) || _("unknown")
-                @record.resize_queue(session[:userid], flavor)
+                # this is always a new request rather than an edit
+                @request_id = nil
+                options = {:src_ids       => [@record.id],
+                           :flavor_name   => flavor.name,
+                           :instance_type => flavor_id}
+                VmCloudReconfigureRequest.make_request(@request_id, options, current_user, true)
```

Comment:
Passing true here for `auto-approve` defeats the purpose of creating the request/task logic. Unless I am missing something you will want to drop the `true` parameter.

Reply:
OK. I misunderstood what was going on here. I added 'true' to allow administrators to initiate this without needing approval, but from the other comments, it sounds like the "admin auto approve" use case is handled via some other means.